src/utils/csv.ts

```diff
@@ -0,0 +1,133 @@
+import { Aggregate, DatapointsGetAggregateDatapoint } from '@cognite/sdk';
+import moment from 'moment-timezone';
+import { getGranularityInSeconds } from './utils';
+
+export interface DatapointsToCSVProps {
+  data: DatapointsGetAggregateDatapoint[];
+  granularity: string;
+  aggregate: Aggregate;
+  delimiter: string;
+  format?: string;
+}
+
+export function datapointsToCSV({
+  data,
+  granularity,
+  aggregate = 'average',
+  delimiter = ',',
```

Comment:
can we use enum with a value instead maybe?
```enum Direction {
    Coma = ',',
    Space = ' ',
    Tab = "\t"
}```

Reply:
done